### ISSUE TITLE
Can not send NEO from Ledger Nano + Neon Wallet

### ISSUE THREAD

**Issue by Cryptomanifest** [CityOfZion/neon-wallet#1045]
Hello Community

Who else is having this issue?

Ok, so I have all the latest versions running on everything. Ledger , Firmware, NEO app, and updated to the Neon Wallet 0.2.3 , on the same computer I have always used for Crypto Only. So I'm not a "newbie" , have been using this wallet combo since it came out without issues.  

I have now tried it on multiple machines to be sure, as well as restoring the old ledger & seed on a new ledger nano s.  Turned Firewall off to see if that would do it, but no solutions

Every time I try to send NEO from wallet to exchange , it says = Transaction Failed : Network Error

And when I attempt to claim GAS its says failed, only sign tx 1 of 2 (2 of 2 never shows up) - same error every time : Calculating Claimable GAS Failed

I need to move my coins. 

I have never had this problem before, no settings have changed.

**Comment by drptbl** [comment]
GAS claiming issue on Ledger should be fixed in next wallet release 👍.

**Comment by Cryptomanifest** [comment]
Lets hope I can send NEO too :) Will look for that release

**Comment by mhuggins** [comment]
This issue should be fixed in [version 0.2.4](https://github.com/CityOfZion/neon-wallet/releases/tag/0.2.4).

**Comment by Cryptomanifest** [comment]
None of this was fixed with 0.2.4 for me

For some reason I can not claim gas, or send NEO or GAS from the wallet. 

I have updated everything 10 times, reinstalled, etc. Can definitely use the ledger for other coins, ran test sends. But every time I log into Neon Wallet , it shows me my balances, but I can not send, always Transaction Failed : Network Error & when I try to claim gas = Calculating Claimable GAS Failed

Running same firewall & antivirus I always run, have never had any issues using Neon + Ledger, now it wont work no matter what I do. Tried another Ledger too, all updated as well. I'm at a loss as far as how to proceed. I need to send some coins and I cant
